src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/ApplicationDesignerView.vb

```diff
@@ -970,6 +970,11 @@ Namespace Microsoft.VisualStudio.Editors.ApplicationDesigner
             Return True
         End Function
 
+        Public Sub AppDesignerAlreadyLoaded()
```

Comment:
Could this be [`Friend`](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/modifiers/friend) rather than `Public`?

Reply:
It looks to me like it could be `Private`, unless there is some reason a method outside this class would need to call it?

Reply:
It's being called by the `ApplicationDesignerWindowPane`, so I think it makes sense changing it to `Friend`.